packages/funnel/src/Funnel.tsx

```diff
@@ -23,6 +23,7 @@ const InnerFunnel = <D extends FunnelDatum>({
     spacing = svgDefaultProps.spacing,
     shapeBlending = svgDefaultProps.shapeBlending,
     valueFormat,
+    valueFormatter,
```

Comment:
I think this should be added to the documentation, we might also want to add a story about it, or to make it available via the playground.

Reply:
Some tests would also be nice.